packages/frontend-shared/src/gql-components/error/BaseError.vue

```diff
@@ -148,6 +149,13 @@ const markdownTarget = ref()
 const baseError = computed(() => props.gql)
 const { markdown } = useMarkdown(markdownTarget, computed(() => props.gql.errorMessage), { classes: { code: ['bg-error-200'] } })
 
+let retrying = false
```

Comment:
This should probably be a `ref`, otherwise you won't get any reactivity out of it and the state will get wiped out on rerender

```
const retrying = ref(false)
...
retrying.value = true
```

Reply:
Good call 👍🏻 still learning about Vue and reactivity stuff